lib/diplomat/policy.rb

```diff
@@ -0,0 +1,112 @@
+module Diplomat
+  # Methods for interacting with the Consul ACL Policy API endpoint
+  class Policy < Diplomat::RestClient
+    @access_methods = %i[list read create delete update]
+    attr_reader :id, :type, :acl
+
+    # Read ACL policy with the given UUID
+    # @param id [String] UUID of the ACL policy to read
+    # @param options [Hash] options parameter hash
+    # @return [Hash] existing ACL policy
+    # rubocop:disable PerceivedComplexity
+    def read(id, options = {}, not_found = :reject, found = :return)
```

Comment:
It's conventional for the options hash to be the last parameter. However, I'm not familiar w/ how this class is used. Is this the expected method signature?